pkg/testsuite/mock/wallet_transactions.go

```diff
@@ -424,21 +423,31 @@ func (w *Wallet) ClaimValidatorRewards(transactionName string, inputName string)
 		panic(fmt.Sprintf("failed to calculate reward for output %s: %s", inputName, err))
 	}
 
-	signedTransaction := w.createSignedTransactionWithOptions(
+	accountOutput := builder.NewAccountOutputBuilderFromPrevious(inputAccount).
+		RemoveFeature(iotago.FeatureStaking).
+		Mana(input.StoredMana() + rewardMana).
```

Comment:
This doesn't take decay and potential Mana into account. It could cause transaction to fail in some cases if the stored Mana was large and it was decayed significantly.